setup.py

```diff
@@ -1,15 +1,20 @@
 import os
 import subprocess
 import warnings
-from packaging import version
-import pip
+from setuptools._vendor.packaging import version
```

Comment:
I would suggest to rollback this change and (if necessary) patch on the spack side. Ideally we can somehow fix the issue mentioned above in the spack package.

```suggestion
from packaging import version
```

Reply:
The change has been rolled back in 5b35c3cc02677ed198d57219415bdd639d9d1907